src/app-contracts/contracts/PullRequests.sol

```diff
@@ -0,0 +1,120 @@
+pragma solidity^0.4.6;
+import "oraclize/usingOraclize.sol";
+
+contract PullRequests is usingOraclize {
+
+    string repo;
+
+    mapping (uint => string) pullRequestIdToCreator;
+    mapping (uint => bool) pullRequestIdToMergedState;
+    mapping (bytes32 => UserQuery) userQueries;
+    mapping (bytes32 => MergedQuery) mergedQueries;
+
+    function PullRequests(string repo_) {
+        repo = repo_;
+    }
+
+    function creators(uint pullRequestId) constant returns (string creator) {
```

Comment:
function `creators` (plural) returns string `creator` (singular) ?

Reply:
Yes, I tried to mimic the auto-generated getter function for mappings, which follow this pattern. But now that you mention it, it is confusing. I will change that.